INSTALL.md

```diff
@@ -1,30 +1,72 @@
+Dependencies](#additional-dependencies).
 
 > **Windows users:** We now have a [YouTube video tutorial](https://www.youtube.com/watch?v=2LApclXFqsg)
 for installing and getting started with the balena CLI on Windows!
 
-## Standalone Installation
+## Executable Installer
 
-1. Download the latest zip file for your OS from https://github.com/balena-io/balena-cli/releases.  
-   (Note that "[Darwin](https://en.wikipedia.org/wiki/Darwin_(operating_system))" is the
-   appropriate zip file for macOS.)
-2. Extract the zip file contents to any folder you choose. The extracted contents will include a
-   `balena-cli` folder.
-3. Add the `balena-cli` folder to the system's `PATH` environment variable. See instructions for:
-   [Windows](https://www.computerhope.com/issues/ch000549.htm) |
-   [Linux](https://stackoverflow.com/questions/14637979/how-to-permanently-set-path-on-linux-unix) |
-   [macOS](https://www.architectryan.com/2012/10/02/add-to-the-path-on-mac-os-x-mountain-lion/#.Uydjga1dXDg)
+_Please note: the executable installers are in **beta** status (recently introduced)._
+
+1. Download the latest installer from the [releases page](https://github.com/balena-io/balena-cli/releases).
+   Look for a file name that ends with the word "installer", for example:  
+   `balena-cli-v10.13.6-windows-x64-installer.exe`  
+   `balena-cli-v10.13.6-macOS-x64-installer.pkg`
+2. Double click to run. Your system may raise a pop-up warning that the installer is from an
+   "unknown publisher" or "unidentified developer". Check the following instructions for how
+   to get through the warnings:
+   [Windows](https://github.com/balena-io/balena-cli/issues/1250) or
+   [macOS](https://github.com/balena-io/balena-cli/issues/1251).
+   (We are looking at how to get the installers digitally signed to avoid the warnings.)
 
-Check that the installation was successful by opening or re-opening a command terminal window
-(so that the PATH environment variable changes take effect), and running these commands:
+After the installation completes, close and re-open any open command terminal windows so that the
+changes made by the installer to the PATH environment variable can take effect. Check that the
+installation was successful by running these commands:
 
 * `balena` - should print the balena CLI help
 * `balena version` - should print the installed CLI version
 
+> Note: If you had previously installed the CLI using a standalone zip package, it may be a good
+> idea to check your system's `PATH` environment variable for duplicate entries, as the terminal
+> will use the entry that comes first. Check the [Standalone Zip Package](#standalone-zip-package)
+> instructions for how to modify the PATH variable.
+
+For your information only, by default the balena CLI is installed to the following folders:  
+
+OS  | Folders
+--- | ---
+Windows: | `C:\Program Files\balena-cli\`  
+macOS:   | `/usr/local/lib/balena-cli/`
+"        | `/usr/local/bin/balena`
```

Comment:
what is the third row? linux?

Reply:
Right -- if you had to ask the question, then clearly I need to change this! :-D
Markdown doesn't seem to allow line breaks inside a table cell, nor a cell to span multiple rows, but both paths applied to macOS. So I used two rows for macOS, and used the double quote as symbol to mean "idem" or "ditto". Now that's a funny thing: thinking about it, the double quote to represent "idem" comes from handwriting -- when I was student, it was common for teachers / lecturers to use a symbol that looks like a double quote when writing rows / tables on a blackboard with chalk. But I guess we don't see it much in the digital world, and I don't know if it's also a regional thing, like more common in some countries than others... :-)
Anyway, I'll reformat the text to avoid a Markdown table, I think.

Reply:
I was able to use the `<br>` tag inside a markdown table cell to achieve the line break.

Reply:
ha, Wikipedia knows about the "ditto mark"! https://en.wikipedia.org/wiki/Ditto_mark
:-)